spark-print-mgr/printers/drivers/driverConfig.json

```diff
@@ -20,6 +20,15 @@
         },
         {
             "type"        : "serial",
+            "blackBox"    : false,
+            "description" : "OverLord",
+            "VID"         : 0403,
```

Comment:
In JSON the 0403 is treated as string literal, this is an oddity of JSON. So 403 will work fine but 0403 is not. This code will not run. Try and start the server "node server.js" and get it running before re-submitting